components/microsoft_outlook/microsoft_outlook.app.mjs

```diff
@@ -1,11 +1,63 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "microsoft_outlook",
-  propDefinitions: {},
+  propDefinitions: {
+    recipients: {
+      label: "Recipients",
+      description: "Array of email addresses",
+      type: "string[]",
+    },
+    subject: {
+      label: "Subject",
+      description: "Subject of the email",
+      type: "string",
+    },
+    content: {
+      label: "Content",
+      description: "Content of the email in text format",
+      type: "string",
+      optional: true,
+    },
+    name: {
+      label: "File name",
+      description: "Name of the file seen by the recipients",
+      type: "string[]",
+      optional: true,
+    },
+    mimetype: {
+      label: "File mimetype",
+      description: "[MIME type as defined by the IANA](https://www.iana.org/assignments/media-types/media-types.xhtml)",
```

Comment:
it might be helpful to include an example (I'm not sure which field from that link I should include here), or what the purpose of including the `mimetype` is.

Reply:
eg: application/pdf